python_modules/libraries/dagster-fivetran/dagster_fivetran/utils.py

```diff
@@ -84,6 +76,18 @@ def metadata_for_table(
     return metadata
 
 
+def get_column_schema_for_columns(columns: Mapping[str, Any]):
```

Comment:
We are moving the logic to extract the column schema from the column details to its own function. This is used in the legacy code and new code as it is in utils, so we keep it as-is.